Dev > Prod (#4463)
postmanlabs/postman-docs: src/pages/docs/collaborating-in-postman/working-with-your-team/collaborating-in-team-workspaces.md

## Next steps

> After learning how to collaborate in team workspaces, you might want more information about managing and working with workspaces:

Reviewer: ```suggestion
After learning how to collaborate in team workspaces, you can learn about managing and working with workspaces:
```

Reviewer: Resolved in TW-811
